### PR TITLE
Implement fine-grained logging for 5 Alluxio scopes + 10 file system commands

### PR DESCRIPTION
**Pull request by abenaru-zz**
# Pull Request Merge Checklist
           
Task backlog identifier related to this PR: [212](https://github.com/s1mbi0se/simbiose_knowledge_base/issues/212)

---

## As a DEVELOPER, I confirm that this PR **doesn't contain**:
- [x] Code lines with syntax errors
- [x] Methods and variables names that aren't clear or doesn't mean anything
- [x] Code that is out of the standard (Example: **PEP-8** for Python code)
- [x] Files with **imports** errors or without any utility
- [x] **TODOs** left behind
- [x] Tests that are being skipped
- [x] Methods placed on incorrect files or folders (Example: methods that are local, but should be placed on /utils)
- [x] Functions, methods or API responses that are publicly returning sensitive information

## As a DEVELOPER, I confirm that I:
- [x] Followed the Simbiose's standard for branch and commit naming
- [x] Concluded the task and achieved the goals according to the task planning I defined before starting to code
- [x] Added all necessary database changes to the [SQL Migrations](https://github.com/s1mbi0se/sql-migrations)
- [x] Created tests for every new flow and method on this Pull Request
- [x] Added the necessary DEBUG logs
- [x] Checked my code using the Simbiose's Corner Cases Guide
- [x] Merged the **develop** branch to this PR/branch before sending the PR for review
- [x] Solved all possible merge conflicts before sending the PR for review
- [x] Corrected all the problems and code smells identified by the quality tools
- [x] Manually executed the necessary tests locally to make sure my changes didn't break anything
- [x] All unitary and integration tests ran by the CI system are passing
- [x] Check the performance of my changes or additions and they didn't make the system slower (I profiled the code)
- [x] Updated the necessary documentation (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Made the self-review of the code before sending the PR for review
- [x] Sent this PR for a peer-review
- [x] Made all the corrections appointed during the peer-review
- [x] Confirmed that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block my merge

---  

## As the TEAM MANAGER, I confirm that I:
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked if the purpose of the task was reached with this PR
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if all related PRs were created when other components were affected by this task (e.g. database changes on [SQL Migrations](https://github.com/s1mbi0se/sql-migrations))
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s pattern for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

---  

## As the QUALITY RESPONSIBLE, I confirm that I:
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s code standard for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

## Important links:
1. [Simbiose Quality Guidelines](https://github.com/s1mbi0se/company/wiki/Quality-Topics#table-of-contents)
2. [Metadata Database Modeling](https://docs.google.com/spreadsheets/d/1Y8_k5xV86kTjgfjbi-PGsJcubUKhWlT2x8ecpnMlyR0/edit?usp=sharing)
3. [Simbiose's Corner Cases Guide](https://www.simbioseventures.com/documentos-e-guias/procedimentos-e-checklists/corner-cases-guide)

### CODE REVIEW

**Review by caiopavanelli — CHANGES_REQUESTED** [review]
Aspect class have missing javadoc. Please add the descriptions.

**Review by caiopavanelli — COMMENTED** [review]
LGTM

**Review by caiopavanelli — APPROVED** [review]

File: logging/src/main/java/alluxio/logging/base/FlowLoggingAspect.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    /**
+     * Starts logging all methods within the white/blacklist once the initial method is called.
+     *
+     * @param point the {@link ProceedingJoinPoint} for the {@code proceed(..)} method
+     *        in AspectJ
+     * @return the same object of the method being wrapped by AspectJ
+     * @throws Throwable the same {@code Throwable} thrown bt the method
+     *         being wrapped by AspectJ
+     */
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/base/FlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/base/FlowLoggingAspect.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    /**
+     * Stops logging once the last method is executed.
+     *
+     * @param point the {@link ProceedingJoinPoint} for the {@code proceed(..)} method
+     *        in AspectJ
+     * @return the same object of the method being wrapped by AspectJ
+     * @throws Throwable the same {@code Throwable} thrown bt the method
+     *         being wrapped by AspectJ
+     */
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/base/FlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeJobMasterFlowLoggingAspect.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class ScopeJobMasterFlowLoggingAspect extends ScopeFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "ScopeJobMasterFlow";
+    private static final String START_METHOD = "execution(* alluxio.master.AlluxioJobMaster.main(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeJobMasterFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeJobMasterFlowLoggingAspect.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeJobMasterFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeJobWorkerFlowLoggingAspect.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class ScopeJobWorkerFlowLoggingAspect extends ScopeFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "ScopeJobWorkerFlow";
+    private static final String START_METHOD = "execution(* alluxio.worker.AlluxioJobWorker.main(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeJobWorkerFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeJobWorkerFlowLoggingAspect.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeJobWorkerFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeMasterFlowLoggingAspect.java [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class ScopeMasterFlowLoggingAspect extends ScopeFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "ScopeMasterFlow";
+    private static final String START_METHOD = "execution(* alluxio.master.AlluxioMaster.main(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeMasterFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeMasterFlowLoggingAspect.java [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeMasterFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeProxyFlowLoggingAspect.java [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class ScopeProxyFlowLoggingAspect extends ScopeFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "ScopeProxyFlow";
+    private static final String START_METHOD = "execution(* alluxio.proxy.AlluxioProxy.main(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeProxyFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeProxyFlowLoggingAspect.java [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeProxyFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeWorkerFlowLoggingAspect.java [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+ */
+@Aspect
+public final class ScopeWorkerFlowLoggingAspect extends ScopeFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "ScopeWorkerFlow";
+    private static final String START_METHOD = "execution(* alluxio.worker.AlluxioWorker.main(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeWorkerFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/ScopeWorkerFlowLoggingAspect.java [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/ScopeWorkerFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsChgrpFlowLoggingAspect.java [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class FsChgrpFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsChgrpFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.ChgrpCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsChgrpFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsChgrpFlowLoggingAspect.java [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsChgrpFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/scope/base/ScopeFlowLoggingAspect.java [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+/**
+ * This class represents every logging aspect for Alluxio scopes (processes),
+ * such as Master, Job Master, Worker, Job Worker, and Proxy.
+ * <p>
+ * This class is used to avoid breaking the D.R.Y. principle - all other logging
+ * aspect classes extending this share the same {@link #WHITE_AND_BLACK_LIST} and
+ * {@link #FINISH_METHOD}.
+ */
+public abstract class ScopeFlowLoggingAspect extends FlowLoggingAspect {
+    // A string representing a list used to determine which methods from
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/base/ScopeFlowLoggingAspect.java]: Use multi-line comments please

File: logging/src/main/java/alluxio/logging/scope/base/ScopeFlowLoggingAspect.java [hunk 16]
```diff
@@ -0,0 +1,24 @@
...
+    // within a given range should be logged or not
+    protected static final String WHITE_AND_BLACK_LIST = "execution(* alluxio..*(..)) && "
+            + "!within(alluxio.logging..*) && "
+            + "!within(br.com.simbiose..*) && "
+            + "!within(java..*)";
+
+    // A string representing the condition/method that needs to be called for this logging
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/scope/base/ScopeFlowLoggingAspect.java]: Use multi-line comments please

File: logging/src/main/java/alluxio/logging/shell/FsChmodFlowLoggingAspect.java [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class FsChmodFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsChmodFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.ChmodCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsChmodFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsChmodFlowLoggingAspect.java [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsChmodFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsChownFlowLoggingAspect.java [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class FsChownFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsChownFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.ChownCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsChownFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsChownFlowLoggingAspect.java [hunk 20]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsChownFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsFreeFlowLoggingAspect.java [hunk 21]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class FsFreeFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsFreeFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.FreeCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsFreeFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsFreeFlowLoggingAspect.java [hunk 22]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsFreeFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsLoadFlowLoggingAspect.java [hunk 23]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class FsLoadFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsLoadFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.LoadCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsLoadFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsLoadFlowLoggingAspect.java [hunk 24]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsLoadFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsLsFlowLoggingAspect.java [hunk 25]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class FsLsFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsLsFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.LsCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsLsFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsLsFlowLoggingAspect.java [hunk 26]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsLsFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsMkdirFlowLoggingAspect.java [hunk 27]
```diff
@@ -0,0 +1,42 @@
...
+@Aspect
+public final class FsMkdirFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsMkdirFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.MkdirCommand.run(..)) || "
+            + "execution(* alluxio.client.file.createDirectory(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsMkdirFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsMkdirFlowLoggingAspect.java [hunk 28]
```diff
@@ -0,0 +1,42 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsMkdirFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsMountFlowLoggingAspect.java [hunk 29]
```diff
@@ -0,0 +1,42 @@
...
+@Aspect
+public final class FsMountFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsMountFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.MountCommand.run(..)) || "
+            + "execution(* alluxio.client.file.mount(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsMountFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsMountFlowLoggingAspect.java [hunk 30]
```diff
@@ -0,0 +1,42 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsMountFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsMvFlowLoggingAspect.java [hunk 31]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class FsMvFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsMvFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.MvCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsMvFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsMvFlowLoggingAspect.java [hunk 32]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsMvFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsPersistFlowLoggingAspect.java [hunk 33]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class FsPersistFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsChgrpFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.PersistCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsPersistFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsPersistFlowLoggingAspect.java [hunk 34]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsPersistFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsRmFlowLoggingAspect.java [hunk 35]
```diff
@@ -0,0 +1,41 @@
...
+ */
+@Aspect
+public final class FsRmFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsRmFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.RmCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsRmFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsRmFlowLoggingAspect.java [hunk 36]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsRmFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsUnmountFlowLoggingAspect.java [hunk 37]
```diff
@@ -0,0 +1,42 @@
...
+@Aspect
+public final class FsUnmountFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsUnmountFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.UnmountCommand.run(..)) || "
+            + "execution(* alluxio.client.file.unmount(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsUnmountFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

File: logging/src/main/java/alluxio/logging/shell/FsUnmountFlowLoggingAspect.java [hunk 38]
```diff
@@ -0,0 +1,42 @@
...
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```
Comment (jpedroantunes) [inline logging/src/main/java/alluxio/logging/shell/FsUnmountFlowLoggingAspect.java]: Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.

**Review by jpedroantunes — CHANGES_REQUESTED** [review]

**Comment by jpedroantunes** [comment]
You forgot to define the "feature" label and to put the "ready" label on it. Please, more attention!

Also, the name of the Pull Request do not follow the Simbiose patterns

**Review by jpedroantunes — APPROVED** [review]
